contracts/IIdentity.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+pragma experimental ABIEncoderV2;
+
+interface IIdentity {
+    function isIdentityDeployer(string memory, address) external returns (bool);
+}
```

Comment:
This is a good solution to reduce the size of the compiled contract. However, this is causing a problem to deploy email.point since this interface is not being automatically copied during the deployment process. The deployment problem could be solved by manually coping the interface to hardhat/contracts folder or it will be solved automatically if this interface is included in point-contracts npm package that is planned to developed. Furthermore, if we decide to use an interface approach to call Identity contract, Identity contract should implement this interface to guarantee that changes in the interface and contract are sync. Another possible improvement is to allow the deployment process accept local dependencies.